betelgeuse.py

```diff
@@ -41,6 +42,7 @@
 # Avoid SSL errors
 ssl._create_default_https_context = ssl._create_unverified_context
 
+INVALID_TEST_PLAN_CHARS_REGEX = re.compile(r'[\\/.:"<>|~!@#$?%^&\'*()+`,= ]')
```

Comment:
This is the same thing as `INVALID_TEST_RUN_CHARS_REGEX`. Since the same regex is needed on more than one place instead of duplicating it it should be renamed to something like `INVALID_CHARS_REGEX` and then refactor the usages of `INVALID_TEST_RUN_CHARS_REGEX` to `INVALID_CHARS_REGEX`.